channels/cl.m3u

```diff
@@ -152,4 +152,9 @@ http://unlimited6-cl.dps.live/vtv/vtv.smil/playlist.m3u8
 #EXTINF:-1 tvg-id="VTVVinaValparaisoHD.cl" tvg-name="VTV Vina - Valparaiso HD" tvg-country="CL" tvg-language="" tvg-logo="http://images.eltelon.com/cache/media/4151/square_150x150-2.jpg" group-title="",VTV Vina - Valparaiso HD
 http://cdn.streamingmedia.cl:1935/live/vtvvina/playlist.m3u8
 #EXTINF:-1 tvg-id="ZappingMusicHD.cl" tvg-name="Zapping Music HD" tvg-country="CL" tvg-language="" tvg-logo="https://i.imgur.com/lHvoKdw.jpg" group-title="Music",Zapping Music HD
-https://zmlive.zappingtv.com/zm_free/zm.smil/playlist.m3u8
+https://zmlive.zappingtv.com/zm_free/zm.smil/playlist.m3u8#EXTINF:-1 tvg-id="TVOSanVicente.cl" tvg-name="TVO San Vicente" tvg-country="CL" tvg-language="Spanish" tvg-logo="https://i.imgur.com/fvnO1X9.png" group-title="",TVO San Vicente
```

Comment:
Please make a new line after playlist.m3u8. In other words separate the link and extinf